.github/ISSUE_TEMPLATE/survey.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Survey template
+about: Issue for tracking work on OTel surveys. 
+title: 'Survey: <name>'
+labels: survey
+assignees: ''
+
+---
+<!-- Please remember to change the title of this issue – change <name> to the name of your survey. -->
+
+### Tasks
+- [ ] Create a new folder in `sig-end-user/end-user-surveys` named as your survey and add a `README.md` file to it
+- [ ] Prepare questions
```

Comment:
In some of the survey issues, I saw that folks use Google Docs to draft questions. Do we want to recommend that or should we leave it open?